Refer to the Docker specific documentation

bintzandt opened the pull request:
Add a link to our Docker specific documentation on the Laravel specific page. This makes it clear that more specific setup is needed in order to get Docker worker with Ray when using Laravel.

Also prefer placement of the `ray.php` configuration file in the `config/` directory instead of in the root of the project. Both seem to work just fine but I think that it is better to stick with Laravel standards when writing documentation, unless there are good reasons not to.

freekmurze commented:
I don't want this by default in the config dir because this configuration is developer specific, and not project specific. Could you revert those changes again? 

Feel free to add an extra paragraph explaining that if every developer on the project needs the same config it's find to put it in the config dir.

bintzandt commented:
I have updated the PR.

Thanks for explaining the reasoning behind using the root directory instead of the `config/` directory. That makes sense. I have added a note about that as well :+1:

freekmurze commented:
Perfect, thanks!
